gensrc/thrift/PlanNodes.thrift

```diff
@@ -108,6 +108,22 @@ enum TFileFormatType {
     FORMAT_ORC,
     FORMAT_JSON,
     FORMAT_PROTO,
+    //csv withnames
+    FORMAT_CSVWITHNAMES_PLAIN,
```

Comment:
may put header and compress type to another message is better

Reply:
Yes, the more reasonable way is to enumerate the file type and compression type separately. However, since the previous logic is that the two are put together, the expansibility is not very good. If you want to separate them, there are many changes involved and the cycle will be relatively long. Therefore, the previous way is still temporarily continued here